Description: Enable "issue_discards" option by default
 to trim SSDs when changing PVs. This option has no effect if the kernel or the
 drive does not support trimming, so it's safe to enable by default.
 See also https://blueprints.launchpad.net/ubuntu/+spec/core-1311-ssd-trimming
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Debian: http://bugs.debian.org/717313

Index: lvm2-2.02.153-1ubuntu1/conf/example.conf.in
===================================================================
--- lvm2-2.02.153-1ubuntu1.orig/conf/example.conf.in
+++ lvm2-2.02.153-1ubuntu1/conf/example.conf.in
@@ -298,7 +298,7 @@
 	# benefit from discards, but SSDs and thinly provisioned LUNs
 	# generally do. If enabled, discards will only be issued if both the
 	# storage and kernel provide support.
-	issue_discards = 0
+	issue_discards = 1
 
 	# Configuration option devices/allow_changes_with_duplicate_pvs.
 	# Allow VG modification while a PV appears on multiple devices.
